tests/review-pr.test.sh

```
@@ -2537,6 +2602,459 @@ assert_grep "$AGENTS_DIR/type-design-analyzer.md" 'detail' \
+assert_grep "$REVIEW_PR" 'project-verification-claims' \
+  "R32b — the step projects one claim card per eligible finding"
+assert_grep "$REVIEW_PR" 'publish-verification' \
+  "R32c — the step publishes the verification sidecar"
+assert_grep "$REVIEW_PR" 'uberdev_child_validate_finding_verifier_result' \
+  "R32d — child results go through the canonical validation boundary, never a hand-rolled parse"
+assert_grep "$REVIEW_PR" 'review_fleet_audit_append' \
+  "R32e — one audit row per verified finding reaches the repo-root audit stream"
+# The threshold is applied controller-side ONLY. The gate's whole design rests
+# on the child never learning the cutoff, so the command must hand the value to
+# publish-verification and to nothing that becomes a prompt.
+assert_grep "$REVIEW_PR" '--threshold "\$REVIEW_CONFIDENCE_THRESHOLD"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)